src/components/containers/tabPanel/TabPanel.js

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { colors } from '../../theme';
+import styled from 'styled-components';
+
+const SelectedButton = styled.button`
+  background-color: ${colors.white};
+  border: 1px solid ${colors.gray};
+  border-bottom: 1px transparent;
+  border-radius: 2px 2px 0 0;
+  color: ${colors.black};
+  display: inline-block;
+  font-size: 18px;
+  font-family: Arial, sans-serif;

```

Comment:
I've removed any font-family refs from other components so that they just inherit the font used by their parent app.